src/Components/Publishing/Typings.ts

```diff
@@ -17,11 +17,7 @@ export enum ArticleAdType {
 export enum AdUnit {
   Desktop_TopLeaderboard = "Desktop_TopLeaderboard",
   Desktop_RightRail1 = "Desktop_RightRail1",
-  Desktop_InContentLB1 = "Desktop_InContentLB1",
```

Comment:
@ashleyjelks are these ad units obsolete?

Reply:
No, they were never used. When I first created this enum I added all of the types in the https://htlbid.com/clients/artsy/index.html document, but they didn't all correspond to the types outlined in the Jira tickets and design specs.